README.md

```
@@ -1,66 +1,106 @@
-- **[byte5](https://byte5.de)**
-- **[OP.GG](https://op.gg)**
+   This will start the Docker containers in detached mode. Sail sets up services like the web server, database, etc., using Docker.
 
-## Contributing
+4. Run the Laravel application:
 
-Thank you for considering contributing to the Laravel framework! The contribution guide can be found in the [Laravel documentation](https://laravel.com/docs/contributions).
+   Open your browser and go to:
 
-## Code of Conduct
+    ```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)